File: csharp/unittest-practice/tests/TortillaMaizTest.cs
using NUnit.Framework;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using unittestPractice.main;

namespace unittestPractice.test
{
    class TortillaMaizTest
    {
        TortillaMaiz _tortillaMaiz;

        [SetUp]
        public void SetUp()
        {
            _tortillaMaiz = new TortillaMaiz();
        }

        [TestCase]
        public void TestCurrentTemperature()
        {
            _tortillaMaiz.SetCurrentTemperature(21);
            Assert.AreEqual(21, _tortillaMaiz.GetCurrentTemperature());
        }

        [TestCase]
        public void TestFalseToast()
        {
            _tortillaMaiz.Toast(false);
            Assert.IsFalse(_tortillaMaiz.IsToasted());
        }

        [TestCase]
        public void TestTrueToast()
        {
            _tortillaMaiz.Toast(true);
            Assert.IsTrue(_tortillaMaiz.IsToasted());
        }

        [TestCase]
        public void TestToasting()
        {
            Assert.AreEqual(20, _tortillaMaiz.GetToastTemperature());
        }
    }
}

File: python/TortillaMaizTest.py
import unittest
from TortillaMaiz import TortillaMaiz


class TortillaHarinaTest(unittest.TestCase):
    tortillaMaiz = TortillaMaiz()

    def testCurrentTemperature(self):
        self.tortillaMaiz.setCurrentTemperature(21)
        self.assertEqual(21, self.tortillaMaiz.getCurrentTemperature())

    def testFalseToast(self):
        self.tortillaMaiz.toast(False)
        self.assertFalse(self.tortillaMaiz.isToasted())

    def testTrueToast(self):
        self.tortillaMaiz.toast(True)
        self.assertTrue(self.tortillaMaiz.isToasted())

    def testToasting(self):
        self.assertEqual(40, self.tortillaMaiz.getToastTemperature())
